<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>501</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>input Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>450</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>output Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutput">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Out2InButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;----</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1144</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionSaveImage"/>
    <addaction name="actionSaveToDB"/>
    <addaction name="actionLoadFromDB"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>화소점</string>
    </property>
    <addaction name="actionBrightnessControl"/>
    <addaction name="actionBinary"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>화소영역</string>
    </property>
    <addaction name="actionMasking"/>
    <addaction name="actionEdgeDetection"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>기하처리</string>
    </property>
    <addaction name="actionResizing"/>
    <addaction name="actionRotation"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>통계처리</string>
    </property>
    <addaction name="actionHistogram"/>
    <addaction name="actionHistogramEqualization"/>
   </widget>
   <widget class="QMenu" name="menuopenCV">
    <property name="title">
     <string>객체탐지</string>
    </property>
    <addaction name="actionObjectDetection"/>
    <addaction name="actionHaarFaceDetection"/>
    <addaction name="actionMaskRCNN"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menuopenCV"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImage"/>
   <addaction name="actionSaveImage"/>
   <addaction name="actionLoadFromDB"/>
   <addaction name="actionSaveToDB"/>
   <addaction name="actionBrightnessControl"/>
   <addaction name="actionMasking"/>
   <addaction name="actionHistogram"/>
   <addaction name="actionHaarFaceDetection"/>
   <addaction name="actionObjectDetection"/>
  </widget>
  <action name="actionOpenImage">
   <property name="icon">
    <iconset>
     <normaloff>:/newPrefix/icon/Folder_01.png</normaloff>:/newPrefix/icon/Folder_01.png</iconset>
   </property>
   <property name="text">
    <string>파일 열기</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="icon">
    <iconset>
     <normaloff>:/newPrefix/icon/FloppyDisk_01.png</normaloff>:/newPrefix/icon/FloppyDisk_01.png</iconset>
   </property>
   <property name="text">
    <string>파일 저장</string>
   </property>
   <property name="iconText">
    <string>파일 저장</string>
   </property>
  </action>
  <action name="actionBrightnessControl">
   <property name="icon">
    <iconset>
     <normaloff>:/newPrefix/icon/Brightness_Control_01.png</normaloff>:/newPrefix/icon/Brightness_Control_01.png</iconset>
   </property>
   <property name="text">
    <string>밝기 조절</string>
   </property>
  </action>
  <action name="actionResizing">
   <property name="text">
    <string>크기 변경</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="icon">
    <iconset>
     <normaloff>:/newPrefix/icon/histogram.png</normaloff>:/newPrefix/icon/histogram.png</iconset>
   </property>
   <property name="text">
    <string>히스토그램</string>
   </property>
  </action>
  <action name="actionRotation">
   <property name="text">
    <string>회전</string>
   </property>
  </action>
  <action name="actionTrainDataToDump">
   <property name="text">
    <string>TrainDataToDump</string>
   </property>
  </action>
  <action name="actionLoadDump">
   <property name="text">
    <string>LoadDump</string>
   </property>
  </action>
  <action name="actionHaarFaceDetection">
   <property name="icon">
    <iconset>
     <normaloff>:/newPrefix/icon/Detection_Person_01.png</normaloff>:/newPrefix/icon/Detection_Person_01.png</iconset>
   </property>
   <property name="text">
    <string>얼굴인식</string>
   </property>
  </action>
  <action name="actionHistogramEqualization">
   <property name="text">
    <string>히스토그램 평활화</string>
   </property>
  </action>
  <action name="actionEmbossing">
   <property name="text">
    <string>엠보싱</string>
   </property>
  </action>
  <action name="actionMasking">
   <property name="icon">
    <iconset>
     <normaloff>:/newPrefix/icon/mask.png</normaloff>:/newPrefix/icon/mask.png</iconset>
   </property>
   <property name="text">
    <string>마스킹</string>
   </property>
  </action>
  <action name="actionEdgeDetection">
   <property name="text">
    <string>엣지추출</string>
   </property>
  </action>
  <action name="actionopenCSV">
   <property name="text">
    <string>openCSV</string>
   </property>
  </action>
  <action name="actionsaveCSV">
   <property name="text">
    <string>saveCSV</string>
   </property>
  </action>
  <action name="actionLoadFromDB">
   <property name="icon">
    <iconset>
     <normaloff>:/newPrefix/icon/DataBase_01.png</normaloff>:/newPrefix/icon/DataBase_01.png</iconset>
   </property>
   <property name="text">
    <string>DB에 저장</string>
   </property>
  </action>
  <action name="actionSaveToDB">
   <property name="icon">
    <iconset>
     <normaloff>:/newPrefix/icon/DataBase_In_01.png</normaloff>:/newPrefix/icon/DataBase_In_01.png</iconset>
   </property>
   <property name="text">
    <string>DB에서 열기</string>
   </property>
  </action>
  <action name="actionBinary">
   <property name="text">
    <string>이진화</string>
   </property>
  </action>
  <action name="action3D">
   <property name="text">
    <string>3D회전</string>
   </property>
  </action>
  <action name="actionObjectDetection">
   <property name="icon">
    <iconset>
     <normaloff>:/newPrefix/icon/ObjectDetection.png</normaloff>:/newPrefix/icon/ObjectDetection.png</iconset>
   </property>
   <property name="text">
    <string>사물인식</string>
   </property>
  </action>
  <action name="actionMaskRCNN">
   <property name="text">
    <string>MaskRCNN</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="PyQt_Resource.qrc"/>
 </resources>
 <connections/>
</ui>
